barretenberg/cpp/src/barretenberg/ultra_honk/decider_verification_key.hpp

```diff
@@ -35,7 +35,7 @@ template <class Flavor, size_t NUM_ = 2> class DeciderVerificationKey_ {
 
     DeciderVerificationKey_() = default;
     DeciderVerificationKey_(std::shared_ptr<VerificationKey> vk)
-        : verification_key(std::move(vk))
```

Comment:
This just is sub-microsecond optimization that we don't need and which hurts clarity here IMO.